x-pack/plugins/security_solution/public/overview/components/detection_response/host_alerts_table/host_alerts_table.tsx

```diff
@@ -0,0 +1,172 @@
+
+const DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID = 'vulnerableHostsBySeverityQuery';
+
+export const HostAlertsTable = React.memo(() => {
+  const { getAppUrl, navigateTo } = useNavigation();
+  const { toggleStatus, setToggleStatus } = useQueryToggle(
+    DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID
+  );
+
+  const { data, isLoading } = useHostAlertsItems({
+    skip: !toggleStatus,
+    queryId: DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID,
+  });
+
+  const columns = useMemo(
+    () => getTableColumns({ getAppUrl, navigateTo }),
+    [getAppUrl, navigateTo]
+  );
+
+  return (
+    <EuiPanel hasBorder data-test-subj="hostSeverityAlertsPanel">
+      <HeaderSection
+        id={DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID}
+        title={i18n.HOST_ALERTS_SECTION_TITLE}
+        titleSize="s"
+        hideSubtitle
+        toggleStatus={toggleStatus}
+        toggleQuery={setToggleStatus}
+      />
+      {toggleStatus &&
+        (isLoading || data.length > 0 ? (
+          <>
+            <EuiBasicTable
+              data-test-subj="hostAlertsTable"
+              columns={columns}
+              items={data}
+              loading={isLoading}
+            />
+            <EuiSpacer size="m" />
+            <EuiButton
+              onClick={() => navigateTo({ url: getAppUrl({ deepLinkId: SecurityPageName.hosts }) })}
+            >
+              {i18n.VIEW_ALL_HOST_ALERTS}
+            </EuiButton>
+          </>
+        ) : (
+          <EuiEmptyPrompt title={<h3>{i18n.NO_ALERTS_FOUND}</h3>} titleSize="xs" />
+        ))}
+    </EuiPanel>
+  );
+});
+
+HostAlertsTable.displayName = 'HostAlertsTable';
+
+const getTableColumns: GetTableColumns = ({ getAppUrl, navigateTo }) => [
+  {
+    field: 'hostName',
+    name: i18n.HOST_ALERTS_HOSTNAME_COLUMN,
+    truncateText: true,
+    textOnly: true,
+    render: (hostName: string) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.hosts, path: hostName });
+
+      return (
+        <EuiLink
+          data-test-subj="userSeverityAlertsTable-hostName"
+          href={`/app/security/hosts/${hostName}`}
+          onClick={(ev?: React.MouseEvent) => {
+            if (ev) {
+              ev.preventDefault();
+            }
+            navigateTo({ url });
+          }}
+        >
+          {hostName}
+        </EuiLink>
+      );
+    },
+  },
+  {
+    field: 'totalAlerts',
+    name: i18n.ALERTS_COLUMN,
+    render: (totalAlerts: number, { hostName }) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.hosts, path: hostName });
+
+      return (
+        <EuiLink
+          data-test-subj="hostSeverityAlertsTable-totalAlerts"
+          href={`/app/security/hosts/${hostName}`}
+          onClick={(ev?: React.MouseEvent) => {
+            if (ev) {
+              ev.preventDefault();
+            }
+            navigateTo({ url });
+          }}
+        >
+          {totalAlerts}
+        </EuiLink>
+      );
+    },
```

Comment:
We already have the host detail link in the host name column, this one would go to `/alerts` with the host filtered, but since it is not clear how Product wants to do that, we can just remove this render function by now.
```suggestion
```

Reply:
removed but keeping the visibility of the count